why.html

```diff
@@ -119,14 +100,14 @@ <h1 data-i18n="why_use_syncloud">Why Use Syncloud?</h1>
 			<div class="tablecel"><div class="tic" data-i18n="data_control">Full Data Control</div><div class="negative" data-i18n="no">No</div></div>
 			<div class="tablecel"><div class="tic" data-i18n="data_protection">Data Protection (Backup possible)</div><div class="positive" data-i18n="yes">Yes</div></div>
 			<div class="tablecel"><div class="tic" data-i18n="open_source_software">Open source software (only)</div><div class="negative" data-i18n="no">No</div></div>
-			<div class="tablecel"><div class="tic" data-i18n="decentralized">Decentralized</div><div class="negative" data-i18n="no">No</div></div>
+			<div class="tablecel"><div class="tic" data-i18n="decentralized">Decentralized</div><div class="positive" data-i18n="yes">Yes</div></div>
 			<div class="tablecel"><div class="tic" data-i18n="easy_to_use">Easy-to-use</div><div class="positive" data-i18n="yes">Yes</div></div>
```

Comment:
Is this true for non-technical users?

Reply:
Are you asking in general or for Synology?

Reply:
Synology

Reply:
I've never used it, just saw a few videos about it. I guess it is easy to setup compared to other home server solutions? At least they are popular.

Reply:
Regarding the "Decentralized" I thought every home server is decentralized.